### DISCUSSION TITLE
VPN Hostpot repeater name & password can't be changed on rooted Android TV!!!

### DISCUSSION THREAD

**Discussion by yhussy** [Mygod/VPNHotspot#232] [Q&A]
Hi, 
1- I have some problems on my rooted "Ugoos AM6", even its rooted and VPN Hostpot is allowed on Magisk as shown in attachment, cant change "repeater" name of password!!! because there is no save icon!!!
2-Can you add Wi-Fi hotspot "Start Wi-Fi  on boot", please?
i attached some screenshots and Android TV device debug information , i hope it well help !!!

![IMAGE 2021-05-30 16:58:27](https://user-images.githubusercontent.com/6484933/120103494-49ded800-c169-11eb-9a75-654774aed73b.jpg)
![IMAGE 2021-05-30 17:05:49](https://user-images.githubusercontent.com/6484933/120103498-4cd9c880-c169-11eb-88b0-890587e04b2f.jpg)

[vpnhotspot-8782258292954601396.log](https://github.com/Mygod/VPNHotspot/files/6566161/vpnhotspot-8782258292954601396.log)

**Comment by Mygod** [comment]
As for 1, you need to update to Android 10 or try different ROMs. As for 2, the tracking issue is #96.
